docs/admin/multiple-schedulers.md

```diff
@@ -0,0 +1,214 @@
+
+### 3. Run the second scheduler in the cluster
+
+In order to run your scheduler in a Kubernetes cluster, just create the deployment specified in the config above in a Kubernetes cluster:
+
+```sh
+$ kubectl create -f my-scheduler.yaml
+```
+
+Verify that the deployment and hence the scheduler pod is running:
+
+```sh
+$ cluster/kubectl.sh get pods --namespace=kube-system
+NAME                                           READY     STATUS    RESTARTS   AGE
+....
+kube-scheduler-lnf4s-4744f                     1/1       Running   0          2m
+...
+```
+
+You should see a "Running" kube-scheduler pod. It is also important to note that the default scheduler managed by Kubernetes runs in a container but not in a pod, so you don't see it in the pods list here.
+
+### 4. Specify schedulers for pods
+
+In order to schedule a given pod using a specific scheduler, we specify the name of the scheduler as an annotation in that pod spec. Let's look at three examples.
+
+
+1. Pod spec without any scheduler annotation
+
+  ```yaml
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    name: no-annotation
+    labels:
+      name: multischeduler-example
+  spec:
+    containers:
+    - name: pod-with-no-annotation-container
+      image: gcr.io/google_containers/pause:2.0
+  ```
+
+  When no scheduler annotation is supplied, the pod is automatically scheduled using the default-scheduler.
+
+  Save this file as `pod1.yaml` and submit it to the Kubernetes cluster.
+
+  ```sh
+  $ kubectl create -f pod1.yaml
+  ```
+2. Pod spec with `default-scheduler` annotation
+
+  ```yaml
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    name: annotation-default-scheduler
+    annotations:
+      scheduler.alpha.kubernetes.io/name: default-scheduler
+    labels:
+      name: multischeduler-example
+  spec:
+    containers:
+    - name: pod-with-default-annotation-container
+      image: gcr.io/google_containers/pause:2.0
+  ```
+
+  A scheduler is specified by supplying the scheduler name as a value to the annotation key `scheduler.alpha.kubernetes.io/name`. In this case, we supply the name of the default scheduler which is `default-scheduler`.
+
+  Save this file as `pod2.yaml` and submit it to the Kubernetes cluster.
+
+  ```sh
+  $ kubectl create -f pod2.yaml
+  ```
+3. Pod spec with `my-scheduler` annotation
+
+  ```yaml
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    name: annotation-second-scheduler
+    annotations:
+      scheduler.alpha.kubernetes.io/name: my-scheduler
+    labels:
+      name: multischeduler-example
+  spec:
+    containers:
+    - name: pod-with-second-annotation-container
+      image: gcr.io/google_containers/pause:2.0
+  ```
+
+  In this case, we specify that this pod should be scheduled using the scheduler that we deployed - `my-scheduler`. Note that the scheduler name in the annotation key should match the name supplied to the scheduler command as an argument in the deployment config for the scheduler.
```

Comment:
s/scheduler name in the annotation key/value of the annotation with key `scheduler.alpha.kubernetes.io/name`/